include/VecRng/RngDefs.h

```diff
@@ -0,0 +1,79 @@
+#ifndef VECRNG_RNGDEFS_H
+#define VECRNG_RNGDEFS_H 1
+
+#include <VecCore/VecCore>
+
+namespace vecRng {
+
+using namespace vecCore;
+
+using ScalarBackend = backend::Scalar;
+#if defined(VECCORE_CUDA)
+using VectorBackend = backend::Scalar;
+#else
+#if defined(VECCORE_ENABLE_VC)
+using VectorBackend = backend::VcVector;
+#elif defined(VECCORE_ENABLE_UMESIMD)
+using VectorBackend = backend::UMESimd;
+#else
+using VectorBackend = backend::Scalar;
+#endif
+#endif
```

Comment:
Within VecCore, the backend cannot be hard-coded. Users must be able to choose what backend to use on their side, including having the choice of using multiple backends at the same time.

Reply:
Again Vecrng is meant to be an independent package using VecCore and it has its
own backend  definition which may be dropped later depending on how the structure
can be reorganized later - currently there is an external test package (VecTest) which used
definitions defined inside RngDefs.h.   For the time being, let's take this as temporarily too.